app/views/users/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<body id="settings-page">
+  <div class="row">
+    <div class="container">
+      <h1>Account Settings</h1>
+      <div class="panel panel-default">
+        <div class="panel-heading">Linked Accounts:</div>
+          <div class="panel-body">
+            <% @user.identities.each do |i| %>
+              <p>
+                <% if i.provider == "twitter" %>
+                  <%= fa_icon("twitter lg") %> Twitter
+                <% elsif i.provider == "vimeo" %>
+                  <%= fa_icon("vimeo lg") %> Vimeo
+                <% else %>
+                  <%= i.provider.capitalize %>
+                <% end %>
+
+                <% if @user.identities.length <= 1 %>
+                  <%= link_to "Remove", deauthorize_path(@user.id, i.provider), method: :put, class: "btn btn-danger btn-sm", data: { confirm: "TBD" } %>
+                <% else %>
+                  <%= link_to "Remove", deauthorize_path(@user.id, i.provider), method: :put, class: "btn btn-danger btn-sm", data: { confirm: "Are you sure? This will remove your ability to log on with this account." } %>
```

Comment:
What is the difference between the actions of these two links in the if/else conditional?